### PR TITLE
Docs link

### PR DESCRIPTION
**Pull request by dashaluna**

### CODE REVIEW

File: includes/admin/class-admin.php [hunk 1]
```diff
@@ -164,8 +167,12 @@ public function display_settings_page() {
 			<h2><?php echo esc_html( $this->page_title ); ?></h2>
 			<p>
 				<?php
-				/* translators: TOC is table of contents */
-				esc_html_e( 'Specify default settings for HM Content TOC plugin', 'hm-content-toc' );
+				/* translators: TOC is table of contents. 1: Opening documentation link tag 2: Closing documentation link tag */
+				printf(
+					esc_html__( 'Specify default settings for HM Content TOC plugin. For more information and usage %1$ssee documentation on github website%2$s.', 'hm-content-toc' ),
+					'<a href="' . esc_url( self::GITHUB_DOC_URL ) . '" target="_blank">',
+					'</a>'
```
Comment (dashaluna) [inline includes/admin/class-admin.php]: @johnbillion not sure if placeholders are fine here? Is there a better solution for this case, where I want to make part of the sentence as a link?

Is it better to use something like this (example from another project):

``` php
// Register text for translation, allow only HTML element `a` with `href` attribute
echo wp_kses(
  sprintf(
    __( 'Try searching for what you were looking for above, or head back to <a href="%s">our homepage</a>.', 'domain' ),
    esc_url( home_url() )
  ),
  array( 'a' => array( 'href' => array() ) )
);
```

Could you explain a bit more which approach is better and why :) thank you!

Comment (johnbillion) [inline includes/admin/class-admin.php]: Using placeholders for the opening and closing `<a>` tags isn't a good idea. It's not clear to the translator what it is, and it's generally a bit rubbish. Your example above is spot on. :+1:

**Comment by dashaluna** [comment]
@johnbillion review & merge

**Comment by dashaluna** [comment]
@johnbillion review & merge

**Pull request merged**